common/gateway-stats-storage/migrations/20241017120000_create_initial_tables.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
given the rest of your code is under GPL-3.0, I guess that's wrong license too

Reply:
Fixed in 6486609